cmd/webhook_v2.go

```diff
@@ -120,7 +122,7 @@ You need to implement the conversion.Hub and conversion.Convertible interfaces f
 
 // webhookOptions represents commandline options for scaffolding a webhook.
 type webhookV2Options struct {
-	res        *resource.Resource
+	resOpts    *resource.Options
 	defaulting bool
```

Comment:
Would not make more sense it has the model instead of Options?

Reply:
What do you mean?
1. Renaming `resource.Options` to `resource.Model`?
2. Using `pkg/model.Resource`?

Reply:
But would make more sense the webhookV2Options received the flags or the model?

> // webhookOptions represents commandline options for scaffolding a webhook.

Yep, I agree the Options shows more appropriated to be here.